templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -13,22 +13,14 @@ source .github/workflows/scripts/utils.sh
 export POST_SCRIPT=$PWD/.github/workflows/scripts/post_script.sh
 export FUNC_TEST_SCRIPT=$PWD/.github/workflows/scripts/func_test_script.sh
 
-# Needed for both starting the service and building the docs.
+# Needed for starting the service
 # Gets set in .github/settings.yml, but doesn't seem to inherited by
 # this script.
 export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
 export PULP_SETTINGS=$PWD/.ci/ansible/settings/settings.py
 
 export PULP_URL="{{ pulp_scheme }}://pulp"
 
-if [[ "$TEST" = "docs" ]]; then
-  if [[ "$GITHUB_WORKFLOW" == "{{ plugin_app_label | camel | default("Pulp") }} CI" ]]; then
-    towncrier build --yes --version 4.0.0.ci
```

Comment:
I wish we could keep this command somewhere.

Reply:
The `towncrier build` command?
On a first sight I just though it was not strictly related to the docs building process, but I can see we want to test if the PR contain valid assets for generating the changelogs.

I'll add a step in the new `docs.yml` called `Build changelog`, wdyt?

Reply:
That sounds great. We should be using the generated changelog in the markdown build (or at least do a syntax check).